ansible/vmware-deploy-testenv.yml

```diff
@@ -37,3 +37,11 @@
   roles:
   - role: vmware-vm
     action: deploy
+
+- hosts: testbed
```

Comment:
When we will use controller in tests we should change destination `hosts` from `testbed` to `testenv`. Maybe we should add this to backlog and put a `TODO` comment here?